drivers/bridge/link.go

```diff
@@ -31,7 +31,8 @@ func newLink(parentIP, childIP string, ports []types.TransportPort, bridge strin
 }
 
 func (l *link) Enable() error {
-	// -A == iptables append flag
+	// -A == iptables append flag, -I = iptables insert flag
+	iptables.OnReloaded(func() { linkContainers("-A", l.parentIP, l.childIP, l.ports, l.bridge, false) })
```

Comment:
Minor: Instead of maintaining two equal statements `linkContainers(...)` I'd prefer defining a function, pass it to iptables.OnReloaded(...) and invoke it at return. In case it needs to change, we need the change in one place only. I feel readability would also improve.